tests/lib/rules/match-component-file-name.js

```diff
@@ -792,9 +1093,21 @@ ruleTester.run('match-component-file-name', rule, {
       errors: [
         {
           message:
-            'Component name `MyComponent` should match file name `my-component`.'
+            'Component name `MyComponent` should match file name `my-component`.',
+          suggestions: [
+            {
+              desc: 'Rename component to match file name.',
+              output: `
+        export default {
+          name: 'my-component',
+          render() { return <div /> }
+        }
+      `
+            }
+          ]
         }
-      ]
+      ],
+      output: null
```

Comment:
Remove `output` as it is no longer needed.

Reply:
Very thanks for the review!!

I removed `output`, and run update the document.